sdlib/d/gc/arena.d

```diff
@@ -148,6 +148,35 @@ public:
 		return null;
 	}
 
+	bool resizeLarge(shared(ExtentMap)* emap, Extent* e, size_t size) shared {
+		assert(e !is null, "Extent is null!");
+		assert(e.isLarge(), "Expected a large extent!");
+		assert(e.arenaIndex == index, "Invalid arena index!");
+		assert(isLargeSize(size));
+
+		// Huge is not supported:
+		if (e.isHuge()) {
+			return false;
+		}
+
+		auto computedPageCount = alignUp(size, PageSize) / PageSize;
+		uint newPages = computedPageCount & uint.max;
+
```

Comment:
Remove that empty line. the assert being on its own is quite confusing as it's not obvious that it is just a sanity check on the previous computation (check that there is no truncation going on, which should be impossible considering the allocatable size we have).